cvat/apps/dataset_manager/annotation.py

```diff
@@ -831,7 +831,7 @@ def interpolate(shape0, shape1):
                 for attr in prev_shape["attributes"]:
                     if attr["spec_id"] not in map(lambda el: el["spec_id"], shape["attributes"]):
                         shape["attributes"].append(deepcopy(attr))
-                if not prev_shape["outside"]:
+                if not prev_shape["outside"] or is_skeleton:
```

Comment:
The bug appears when we save skeleton tracks as shapes and some skeleton element has `outside=True`, `keyframe=False`. We don't save such shape if it is, for example, bounding box. But in the case of the skeleton element, we need to save it, otherwise we aren't be able to import these annotations in the future. 
If the previous shape had `outside=True` in the case of the skeleton, we add the interpolated shapes.

Reply:
Oh, I see. Sorry, I misread the original issue.